### PR TITLE
Documentation/zenko 1804 improve dashboard page

### PR DESCRIPTION
**Pull request by wabernat**
This PR adds depth to the Operations Guide's UI descriptions by detailing and providing contextual links and descriptions for the Dashboard's sidebar links. 

This PR fixes #ZENKO-1804

### CODE REVIEW

**Comment by wabernat** [comment]
Pending ZENKO-1817

**Comment by wabernat** [comment]
@bert-e approve

File: docs/docsource/operation/Orbit_UI/Dashboard.rst [hunk 1]
```diff
@@ -8,9 +8,26 @@ On login, the dashboard displays:
...
+Visible from all Orbit windows, the sidebar contains navigational
+buttons for:
+
+   * :ref:`Statistics`
+   * :ref:`Location Status<location_status>`
+   * :ref:`Storage Accounts <add_a_new_user>`
+   * :ref:`Storage Locations <orbit_add_location>`
+   * :ref:`Endpoints <endpoint>`
+   * :ref:`Browser <create_a_bucket>`
+   * :ref:`Search<orbit_md_search>`
```
Comment (jonathan-gramain) [inline docs/docsource/operation/Orbit_UI/Dashboard.rst]: minor: for consistency, you might add a spacing like `Search <orbit_md_search>` here and below.

**Review by jonathan-gramain — APPROVED** [review]

**Review by tmacro — APPROVED** [review]

File: docs/docsource/operation/Orbit_UI/Dashboard.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/Orbit_UI/Dashboard.rst]: done.

**Comment by wabernat** [comment]
I messed up the PR. See: https://github.com/scality/Zenko/pull/713

**Pull request closed without merging**